src/printapp/static/js/app.js

```diff
@@ -140,10 +140,35 @@ App.SignInController = Ember.Controller.extend({
 });
 
 App.QueueController = Ember.ArrayController.extend({
-  itemController: 'queueItem'
+  needs: ['application','queue'],
+  itemController: 'queueItem',
+
+  actions : {
+
+    // TODO: use "item.loading = true" and read that in the view to decide when to show the loading spinner.
+    deleteJob: function(job_id) {
+      var controller = this;
+      var queue = controller.get('model');
+      var documentToDelete = queue.findBy('job_id', job_id);
+      documentToDelete.loading = true;
```

Comment:
I'm not sure what you're asking. I think if you want to update an item in a model, you can't do it directly. You have to use methods of the `ArrayModel`, otherwise Ember won't know that you changed anything.